src/ui/Facet/Facet.ts

```diff
@@ -365,13 +365,15 @@ export class Facet extends Component {
      */
     numberOfValuesInFacetSearch: ComponentOptions.buildNumberOption({ defaultValue: 15, min: 1, section: 'FacetSearch' }),
     /**
-     * Specifies whether [wildcards]{@link AllowedValuesPatternType.wildcards} will be used as the [allowedValuesPatternType]{@link IGroupByRequest.allowedValuesPatternType}
+     * Specifies whether [wildcards]{@link AllowedValuesPatternType.wildcards} are used as the [allowedValuesPatternType]{@link IGroupByRequest.allowedValuesPatternType}
```

Comment:
I believe you will have to capitalize _Wildcards_ in the link:

[`wildcards`]{@link AllowedValuesPatternType.Wildcards}

Reply:
I just talked to Oli. This is a difference of behavior between Listfield and GroupBy.
GroupBy did not use to return results if the search term did not start with the same thing. Now it's fixed. With the previous method, it used to work the same way. That's why I wasn't able to reproduce.